hazelcast/src/main/java/com/hazelcast/query/impl/Indexes.java

```diff
@@ -93,6 +100,10 @@ public void saveEntryIndex(QueryableEntry queryableEntry, Object oldValue) throw
         }
     }
 
+    public boolean isGlobal() {
```

Comment:
can you please add some javaDoc on this method as an answer to questions what is `global` & what is it used for?